web/src/lib/domain-logic/getNavBarBusinessTitle.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { getMergedConfig } from "@/contexts/configContext";
+import { getNavBarBusinessTitle } from "@/lib/domain-logic/getNavBarBusinessTitle";
+import { LookupIndustryById, LookupLegalStructureById } from "@businessnjgovnavigator/shared";
+import { generateProfileData, generateUserData } from "@businessnjgovnavigator/shared/test";
+
+const Config = getMergedConfig();
+
+describe("getNavBarBusinessTitle", () => {
+  it("shows guest text if industryId is undefined", () => {
+    const userData = generateUserData({
+      profileData: generateProfileData({
+        businessName: "",
+        industryId: undefined,
+        legalStructureId: "limited-liability-company",
+      }),
+    });
+    const NavBarBusinessTitle = getNavBarBusinessTitle(userData);
```

Comment:
for this in all these tests, we should always camelCase and not PascalCase variable names

Reply:
Ah yes